Exception in JsonAddonService / marketplace bundle at startup

last snapshot 2694, at startup of OH:
```
2022-01-15 16:13:03.120 [ERROR] [place.internal.json.JsonAddonService] - bundle org.openhab.core.addon.marketplace:3.3.0.202201150311 (152)[org.openhab.core.addon.marketplace.internal.json.JsonAddonService(49)] : Error during instantiation of the implementation object
java.lang.reflect.InvocationTargetException: null
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:?]
        at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:490) ~[?:?]
        at org.apache.felix.scr.impl.inject.internal.ComponentConstructorImpl.newInstance(ComponentConstructorImpl.java:316) ~[bundleFile:?]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.createImplementationObject(SingleComponentManager.java:286) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.createComponent(SingleComponentManager.java:115) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.getService(SingleComponentManager.java:1000) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.getServiceInternal(SingleComponentManager.java:973) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.AbstractComponentManager.activateInternal(AbstractComponentManager.java:776) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.AbstractComponentManager.enableInternal(AbstractComponentManager.java:674) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.AbstractComponentManager.enable(AbstractComponentManager.java:437) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.ConfigurableComponentHolder.enableComponents(ConfigurableComponentHolder.java:667) [bundleFile:?]
        at org.apache.felix.scr.impl.BundleComponentActivator.initialEnable(BundleComponentActivator.java:305) [bundleFile:?]
        at org.apache.felix.scr.impl.Activator.loadComponents(Activator.java:554) [bundleFile:?]
        at org.apache.felix.scr.impl.Activator.access$200(Activator.java:70) [bundleFile:?]
        at org.apache.felix.scr.impl.Activator$ScrExtension.start(Activator.java:421) [bundleFile:?]
        at org.apache.felix.scr.impl.AbstractExtender.createExtension(AbstractExtender.java:196) [bundleFile:?]
        at org.apache.felix.scr.impl.AbstractExtender.modifiedBundle(AbstractExtender.java:169) [bundleFile:?]
        at org.apache.felix.scr.impl.AbstractExtender.modifiedBundle(AbstractExtender.java:49) [bundleFile:?]
        at org.osgi.util.tracker.BundleTracker$Tracked.customizerModified(BundleTracker.java:488) [osgi.core-7.0.0.jar:?]
        at org.osgi.util.tracker.BundleTracker$Tracked.customizerModified(BundleTracker.java:420) [osgi.core-7.0.0.jar:?]
        at org.osgi.util.tracker.AbstractTracked.track(AbstractTracked.java:232) [osgi.core-7.0.0.jar:?]
        at org.osgi.util.tracker.BundleTracker$Tracked.bundleChanged(BundleTracker.java:450) [osgi.core-7.0.0.jar:?]
        at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:945) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.framework.eventmgr.ListenerQueue.dispatchEventSynchronous(ListenerQueue.java:151) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEventPrivileged(EquinoxEventPublisher.java:232) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEvent(EquinoxEventPublisher.java:138) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEvent(EquinoxEventPublisher.java:130) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor.publishModuleEvent(EquinoxContainerAdaptor.java:217) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.Module.publishEvent(Module.java:499) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.Module.start(Module.java:486) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel$2.run(ModuleContainer.java:1849) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor$1$1.execute(EquinoxContainerAdaptor.java:136) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1842) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1785) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1747) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1669) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345) [org.eclipse.osgi-3.16.300.jar:?]
Caused by: java.lang.ClassCastException: class java.lang.String cannot be cast to class java.lang.Boolean (java.lang.String and java.lang.Boolean are in module java.base of loader 'bootstrap')
        at org.openhab.core.addon.marketplace.internal.json.JsonAddonService.remoteEnabled(JsonAddonService.java:273) ~[?:?]
        at org.openhab.core.addon.marketplace.internal.json.JsonAddonService.refreshSource(JsonAddonService.java:141) ~[?:?]
        at org.openhab.core.addon.marketplace.internal.json.JsonAddonService.modified(JsonAddonService.java:116) ~[?:?]
        at org.openhab.core.addon.marketplace.internal.json.JsonAddonService.<init>(JsonAddonService.java:108) ~[?:?]
        ... 42 more
2022-01-15 16:13:13.767 [ERROR] [place.internal.json.JsonAddonService] - bundle org.openhab.core.addon.marketplace:3.3.0.202201150311 (152)[org.openhab.core.addon.marketplace.internal.json.JsonAddonService(49)] : Error during instantiation of the implementation object
java.lang.reflect.InvocationTargetException: null
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:?]
        at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:490) ~[?:?]
        at org.apache.felix.scr.impl.inject.internal.ComponentConstructorImpl.newInstance(ComponentConstructorImpl.java:316) ~[?:?]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.createImplementationObject(SingleComponentManager.java:286) ~[?:?]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.createComponent(SingleComponentManager.java:115) ~[?:?]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.getService(SingleComponentManager.java:1000) ~[?:?]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.getServiceInternal(SingleComponentManager.java:973) ~[?:?]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.getService(SingleComponentManager.java:918) ~[?:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse$1.run(ServiceFactoryUse.java:216) ~[org.eclipse.osgi-3.16.300.jar:?]
        at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse.factoryGetService(ServiceFactoryUse.java:213) ~[org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse.getService(ServiceFactoryUse.java:114) ~[org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceConsumer$2.getService(ServiceConsumer.java:48) ~[org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceRegistrationImpl.getService(ServiceRegistrationImpl.java:547) ~[org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.getService(ServiceRegistry.java:533) ~[org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.BundleContextImpl.getService(BundleContextImpl.java:656) ~[org.eclipse.osgi-3.16.300.jar:?]
        at org.apache.felix.scr.impl.manager.SingleRefPair.getServiceObject(SingleRefPair.java:88) ~[?:?]
        at org.apache.felix.scr.impl.inject.methods.BindMethod.getServiceObject(BindMethod.java:675) ~[?:?]
        at org.apache.felix.scr.impl.manager.DependencyManager.getServiceObject(DependencyManager.java:2556) ~[?:?]
        at org.apache.felix.scr.impl.manager.DependencyManager$MultipleDynamicCustomizer.prebind(DependencyManager.java:431) ~[?:?]
        at org.apache.felix.scr.impl.manager.DependencyManager.prebind(DependencyManager.java:1827) ~[?:?]
        at org.apache.felix.scr.impl.manager.AbstractComponentManager.collectDependencies(AbstractComponentManager.java:1057) ~[?:?]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.getServiceInternal(SingleComponentManager.java:953) ~[?:?]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.getService(SingleComponentManager.java:918) ~[?:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse$1.run(ServiceFactoryUse.java:216) ~[org.eclipse.osgi-3.16.300.jar:?]
        at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse.factoryGetService(ServiceFactoryUse.java:213) ~[org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse.getService(ServiceFactoryUse.java:114) ~[org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceUse.newServiceObject(ServiceUse.java:99) ~[org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceConsumer$1.getService(ServiceConsumer.java:33) ~[org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceRegistrationImpl.getService(ServiceRegistrationImpl.java:547) ~[org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceObjectsImpl.getService(ServiceObjectsImpl.java:92) ~[org.eclipse.osgi-3.16.300.jar:?]
        at org.apache.aries.jax.rs.whiteboard.internal.utils.Utils.lambda$null$7(Utils.java:206) ~[?:?]
        at org.apache.aries.component.dsl.OSGi.lambda$null$68(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.Publisher.apply(Publisher.java:28) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$75(OSGi.java:715) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.JustOSGiImpl.lambda$new$2(JustOSGiImpl.java:47) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$recoverWith$76(OSGi.java:711) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$flatMap$69(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$68(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.JustOSGiImpl.lambda$new$2(JustOSGiImpl.java:47) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.BundleContextOSGiImpl.lambda$new$0(BundleContextOSGiImpl.java:29) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$flatMap$69(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.ChangeContextOSGiImpl.lambda$new$0(ChangeContextOSGiImpl.java:31) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$recoverWith$76(OSGi.java:711) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$flatMap$69(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$effects$65(OSGi.java:596) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$effects$65(OSGi.java:596) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$effects$65(OSGi.java:596) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$68(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$64(OSGi.java:602) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.JustOSGiImpl.lambda$new$2(JustOSGiImpl.java:47) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$68(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$64(OSGi.java:602) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.Publisher.apply(Publisher.java:28) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$29(OSGi.java:296) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.UpdateSupport.deferPublication(UpdateSupport.java:40) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$32(OSGi.java:295) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$64(OSGi.java:602) ~[bundleFile:1.2.2]
        at org.apache.aries.jax.rs.whiteboard.internal.ServiceReferenceFilteredPublisher.publishIfMatched(ServiceReferenceFilteredPublisher.java:55) ~[?:?]
        at org.apache.aries.jax.rs.whiteboard.internal.ServiceReferenceRegistry.lambda$waitFor$1(ServiceReferenceRegistry.java:66) ~[?:?]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$effects$65(OSGi.java:596) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$transform$84(OSGi.java:757) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$effects$65(OSGi.java:596) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$flatMap$69(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$68(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.EffectsOSGi.lambda$new$1(EffectsOSGi.java:35) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$flatMap$69(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$effects$65(OSGi.java:596) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$flatMap$69(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$68(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$64(OSGi.java:602) ~[bundleFile:1.2.2]
        at org.apache.aries.jax.rs.whiteboard.internal.FilteredPublisher.publishIfMatched(FilteredPublisher.java:51) ~[?:?]
        at org.apache.aries.jax.rs.whiteboard.internal.Registry.lambda$waitForService$1(Registry.java:69) ~[?:?]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$effects$65(OSGi.java:596) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$68(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.EffectsOSGi.lambda$new$1(EffectsOSGi.java:35) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$flatMap$69(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$68(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.JustOSGiImpl.lambda$new$2(JustOSGiImpl.java:47) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$flatMap$69(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$68(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.JustOSGiImpl.lambda$new$2(JustOSGiImpl.java:47) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$flatMap$69(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$flatMap$69(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.lambda$create$0(OSGiImpl.java:39) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$68(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.JustOSGiImpl.lambda$new$2(JustOSGiImpl.java:47) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$68(OSGi.java:674) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.Publisher.apply(Publisher.java:28) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$66(OSGi.java:663) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.Pad.publish(Pad.java:59) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.DistributeOSGiImpl.lambda$null$1(DistributeOSGiImpl.java:51) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$64(OSGi.java:602) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.Publisher.apply(Publisher.java:28) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.HighestRankingOSGi.lambda$null$1(HighestRankingOSGi.java:62) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.Pad.publish(Pad.java:59) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$79(OSGi.java:739) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.JustOSGiImpl.lambda$new$2(JustOSGiImpl.java:47) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.OSGiImpl.run(OSGiImpl.java:50) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$80(OSGi.java:732) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$64(OSGi.java:602) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.Publisher.apply(Publisher.java:28) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.OSGi.lambda$null$66(OSGi.java:663) ~[bundleFile:1.2.2]
        at org.apache.aries.component.dsl.Publisher.apply(Publisher.java:28) [bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.ServiceReferenceOSGi$DefaultServiceTrackerCustomizer.addingService(ServiceReferenceOSGi.java:74) [bundleFile:1.2.2]
        at org.apache.aries.component.dsl.internal.ServiceReferenceOSGi$DefaultServiceTrackerCustomizer.addingService(ServiceReferenceOSGi.java:56) [bundleFile:1.2.2]
        at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:943) [osgi.core-7.0.0.jar:?]
        at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:871) [osgi.core-7.0.0.jar:?]
        at org.osgi.util.tracker.AbstractTracked.trackAdding(AbstractTracked.java:256) [osgi.core-7.0.0.jar:?]
        at org.osgi.util.tracker.AbstractTracked.track(AbstractTracked.java:229) [osgi.core-7.0.0.jar:?]
        at org.osgi.util.tracker.ServiceTracker$Tracked.serviceChanged(ServiceTracker.java:903) [osgi.core-7.0.0.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.FilteredServiceListener.serviceChanged(FilteredServiceListener.java:120) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:957) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.framework.eventmgr.ListenerQueue.dispatchEventSynchronous(ListenerQueue.java:151) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.publishServiceEventPrivileged(ServiceRegistry.java:936) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.publishServiceEvent(ServiceRegistry.java:873) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceRegistrationImpl.register(ServiceRegistrationImpl.java:141) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.registerService(ServiceRegistry.java:261) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.BundleContextImpl.registerService(BundleContextImpl.java:496) [org.eclipse.osgi-3.16.300.jar:?]
        at org.apache.felix.scr.impl.manager.AbstractComponentManager$3.register(AbstractComponentManager.java:929) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.AbstractComponentManager$3.register(AbstractComponentManager.java:915) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.RegistrationManager.changeRegistration(RegistrationManager.java:133) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.AbstractComponentManager.registerService(AbstractComponentManager.java:984) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.AbstractComponentManager.activateInternal(AbstractComponentManager.java:752) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.AbstractComponentManager.enableInternal(AbstractComponentManager.java:674) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.AbstractComponentManager.enable(AbstractComponentManager.java:437) [bundleFile:?]
        at org.apache.felix.scr.impl.manager.ConfigurableComponentHolder.enableComponents(ConfigurableComponentHolder.java:667) [bundleFile:?]
        at org.apache.felix.scr.impl.BundleComponentActivator.initialEnable(BundleComponentActivator.java:305) [bundleFile:?]
        at org.apache.felix.scr.impl.Activator.loadComponents(Activator.java:554) [bundleFile:?]
        at org.apache.felix.scr.impl.Activator.access$200(Activator.java:70) [bundleFile:?]
        at org.apache.felix.scr.impl.Activator$ScrExtension.start(Activator.java:421) [bundleFile:?]
        at org.apache.felix.scr.impl.AbstractExtender.createExtension(AbstractExtender.java:196) [bundleFile:?]
        at org.apache.felix.scr.impl.AbstractExtender.modifiedBundle(AbstractExtender.java:169) [bundleFile:?]
        at org.apache.felix.scr.impl.AbstractExtender.modifiedBundle(AbstractExtender.java:49) [bundleFile:?]
        at org.osgi.util.tracker.BundleTracker$Tracked.customizerModified(BundleTracker.java:488) [osgi.core-7.0.0.jar:?]
        at org.osgi.util.tracker.BundleTracker$Tracked.customizerModified(BundleTracker.java:420) [osgi.core-7.0.0.jar:?]
        at org.osgi.util.tracker.AbstractTracked.track(AbstractTracked.java:232) [osgi.core-7.0.0.jar:?]
        at org.osgi.util.tracker.BundleTracker$Tracked.bundleChanged(BundleTracker.java:450) [osgi.core-7.0.0.jar:?]
        at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:945) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.framework.eventmgr.ListenerQueue.dispatchEventSynchronous(ListenerQueue.java:151) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEventPrivileged(EquinoxEventPublisher.java:232) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEvent(EquinoxEventPublisher.java:138) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEvent(EquinoxEventPublisher.java:130) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor.publishModuleEvent(EquinoxContainerAdaptor.java:217) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.Module.publishEvent(Module.java:499) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.Module.start(Module.java:486) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel$2.run(ModuleContainer.java:1849) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor$1$1.execute(EquinoxContainerAdaptor.java:136) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1842) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1785) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1747) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1669) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234) [org.eclipse.osgi-3.16.300.jar:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345) [org.eclipse.osgi-3.16.300.jar:?]
Caused by: java.lang.ClassCastException: class java.lang.String cannot be cast to class java.lang.Boolean (java.lang.String and java.lang.Boolean are in module java.base of loader 'bootstrap')
        at org.openhab.core.addon.marketplace.internal.json.JsonAddonService.remoteEnabled(JsonAddonService.java:273) ~[?:?]
        at org.openhab.core.addon.marketplace.internal.json.JsonAddonService.refreshSource(JsonAddonService.java:141) ~[?:?]
        at org.openhab.core.addon.marketplace.internal.json.JsonAddonService.modified(JsonAddonService.java:116) ~[?:?]
        at org.openhab.core.addon.marketplace.internal.json.JsonAddonService.<init>(JsonAddonService.java:108) ~[?:?]
        ... 207 more
2022-01-15 16:13:14.107 [WARN ] [t.core.internal.addons.AddonResource] - bundle org.openhab.core.io.rest.core:3.3.0.202201150308 (180)[org.openhab.core.io.rest.core.internal.addons.AddonResource(157)] : Could not get service from ref {org.openhab.core.addon.AddonService}={service.config.factory=false, service.id=198, service.bundleid=152, service.scope=bundle, service.config.category=system, service.pid=org.openhab.jsonaddonservice, service.config.label=Json 3rd Party Add-on Service, component.name=org.openhab.core.addon.marketplace.internal.json.JsonAddonService, service.config.description.uri=system:jsonaddonservice, component.id=49}
```

--- a/bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java
+++ b/bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java
@@ -23,6 +23,7 @@ import java.net.URLConnection;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Date;
+import java.util.Dictionary;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
@@ -47,6 +48,7 @@ import org.openhab.core.addon.marketplace.internal.community.model.DiscourseCate
 import org.openhab.core.addon.marketplace.internal.community.model.DiscourseCategoryResponseDTO.DiscourseUser;
 import org.openhab.core.addon.marketplace.internal.community.model.DiscourseTopicResponseDTO;
 import org.openhab.core.addon.marketplace.internal.community.model.DiscourseTopicResponseDTO.DiscoursePostLink;
+import org.openhab.core.config.core.ConfigParser;
 import org.openhab.core.config.core.ConfigurableService;
 import org.openhab.core.events.Event;
 import org.openhab.core.events.EventPublisher;
@@ -85,6 +87,7 @@ public class CommunityMarketplaceAddonService implements AddonService {
     static final String CONFIG_URI = "system:marketplace";
     static final String CONFIG_API_KEY = "apiKey";
     static final String CONFIG_SHOW_UNPUBLISHED_ENTRIES_KEY = "showUnpublished";
+    private static final String CFG_REMOTE = "remote";
 
     private static final String COMMUNITY_BASE_URL = "https://community.openhab.org";
     private static final String COMMUNITY_MARKETPLACE_URL = COMMUNITY_BASE_URL + "/c/marketplace/69/l/latest";
@@ -479,11 +482,12 @@ public class CommunityMarketplaceAddonService implements AddonService {
     private boolean remoteEnabled() {
         try {
             Configuration configuration = configurationAdmin.getConfiguration("org.openhab.addons", null);
-            if (configuration.getProperties() != null) {
-                return (boolean) Objects.requireNonNullElse(configuration.getProperties().get("remote"), true);
-            } else {
+            Dictionary<String, Object> properties = configuration.getProperties();
+            if (properties == null) {
+                // if we can't determine a set property, we use true (default is remote enabled)
                 return true;
             }
+            return ConfigParser.valueAsOrElse(properties.get(CFG_REMOTE), Boolean.class, true);
         } catch (IOException e) {
             return true;
         }

--- a/bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/json/JsonAddonService.java
+++ b/bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/json/JsonAddonService.java
@@ -21,6 +21,7 @@ import java.net.URL;
 import java.net.URLConnection;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Dictionary;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
@@ -39,6 +40,7 @@ import org.openhab.core.addon.AddonType;
 import org.openhab.core.addon.marketplace.MarketplaceAddonHandler;
 import org.openhab.core.addon.marketplace.MarketplaceHandlerException;
 import org.openhab.core.addon.marketplace.internal.json.model.AddonEntryDTO;
+import org.openhab.core.config.core.ConfigParser;
 import org.openhab.core.config.core.ConfigurableService;
 import org.openhab.core.events.Event;
 import org.openhab.core.events.EventPublisher;
@@ -79,6 +81,7 @@ public class JsonAddonService implements AddonService {
 
     private static final String CONFIG_URLS = "urls";
     private static final String CONFIG_SHOW_UNSTABLE = "showUnstable";
+    private static final String CFG_REMOTE = "remote";
 
     private static final Map<String, AddonType> TAG_ADDON_TYPE_MAP = Map.of( //
             "automation", new AddonType("automation", "Automation"), //
@@ -269,11 +272,12 @@ public class JsonAddonService implements AddonService {
     private boolean remoteEnabled() {
         try {
             Configuration configuration = configurationAdmin.getConfiguration("org.openhab.addons", null);
-            if (configuration.getProperties() != null) {
-                return (boolean) Objects.requireNonNullElse(configuration.getProperties().get("remote"), true);
-            } else {
+            Dictionary<String, Object> properties = configuration.getProperties();
+            if (properties == null) {
+                // if we can't determine a set property, we use true (default is remote enabled)
                 return true;
             }
+            return ConfigParser.valueAsOrElse(properties.get(CFG_REMOTE), Boolean.class, true);
         } catch (IOException e) {
             return true;
         }

